dev/fattest.simplicity/src/componenttest/containers/ExternalTestServiceDockerClientStrategy.java

```diff
@@ -391,11 +398,62 @@ private static boolean useRemoteDocker() {
         } while (false);
 
         reason = result ? //
-                        "Running against remote docker host.  Reason: " + reason : //
+                        "Running against remote docker host. Reason: " + reason : //
                         "Running against local docker host. Reason: " + reason;
 
         Log.info(c, "useRemoteDocker", reason);
         return result;
+    }
 
+    /**
+     * Determines if we are going to use the Artifactory name substitutor.
+     *
+     * Priority:
+     * 1. System Property: fat.test.use.artifactory.substitution
+     *
+     * default (USE_REMOTE_DOCKER_HOST)
+     *
+     * NOTE: There are situations were we will still decide NOT to apply the
+     * substitution for synthetic images, and programmatically committed images.
+     * This will be determined on an image-to-image basis
+     *
+     * @see    ArtifactoryImageNameSubstitutor#apply(org.testcontainers.utility.DockerImageName)
+     *
+     * @return true, we are using the substitutor, false otherwise.
+     */
+    private static boolean useArtifactorySubstitutor() {
+        boolean result;
+        String reason;
+
+        do {
+            //State 1: fat.test.use.artifactory.substitution should always be honored first
+            if (System.getProperty("fat.test.use.artifactory.substitution") != null) {
+                result = Boolean.getBoolean("fat.test.use.artifactory.substitution");
+                reason = "fat.test.use.artifactory.substitution set to " + result;
+
+                if (result == false && USE_REMOTE_DOCKER_HOST == true) {
+                    Log.warning(c, reason + System.lineSeparator()
+                                   + "Based on a priority system we decided to use a remote docker host for testing. "
+                                   + "Therefore, we cannot honor the request to NOT use artifactory. "
+                                   + "To resolve this issue either remove the fat.test.use.artifactory.substitution property, "
+                                   + "or force this test to use the a local docker host using fat.test.use.remote.docker=true.");
```

Comment:
I think this might want to be fat.test.use.remote.docker=false,

Setting remote.docker=true to use a local docker makes little sense